doc/user_guide/udf-6.sql

```diff
@@ -0,0 +1,109 @@
+open schema "MY_SCHEMA";
+
+ALTER SYSTEM SET SCRIPT_LANGUAGES='R=builtin_r JAVA=builtin_java PYTHON3=builtin_python3 PYTHON3_AAF=localzmq+protobuf:///bfsdefault/default/temp/exasol_advanced_analytics_framework_container_release?lang=python#/buckets/bfsdefault/default/temp/exasol_advanced_analytics_framework_container_release/exaudf/exaudfclient_py3';
+
+--/
+CREATE OR REPLACE PYTHON3_AAF SET SCRIPT "MY_SCHEMA"."MY_QUERY_HANDLER_UDF"(...)
+EMITS (outputs VARCHAR(2000000)) AS
+
+from typing import Union
+from exasol_advanced_analytics_framework.udf_framework.udf_query_handler import UDFQueryHandler
+from exasol_advanced_analytics_framework.query_handler.context.query_handler_context import QueryHandlerContext
+from exasol_advanced_analytics_framework.query_result.query_result import QueryResult
+from exasol_advanced_analytics_framework.query_handler.result import Result, Continue, Finish
+from exasol_advanced_analytics_framework.query_handler.query.select_query import SelectQuery, SelectQueryWithColumnDefinition
+from exasol_data_science_utils_python.schema.column import Column
+from exasol_data_science_utils_python.schema.column_name import ColumnName
+from exasol_data_science_utils_python.schema.column_type import ColumnType
+from datetime import datetime
+from io import BytesIO, TextIOWrapper
+
+
+# proposal write parameter into temp table in schema ...
+class CustomQueryHandler(UDFQueryHandler):
+    def __init__(self, parameter: str, query_handler_context: QueryHandlerContext):
+        super().__init__(parameter, query_handler_context)
+        self.parameter = parameter
+        self.query_handler_context = query_handler_context
+        self.bfs_proxy = query_handler_context.get_temporary_bucketfs_location()
+        self.db_table_proxy = query_handler_context.get_temporary_table_name()
+
+    @property
+    def _bfs_temp_file(self):
+        return self.bfs_proxy.bucketfs_location() / "temp_file.txt"
+
+    def _sample_content(self, key: str) -> str:
+        timestamp = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+        return f"{timestamp} {key} {self.parameter}"
+
+    def _write_to_bucketfs(self):
+        self._bfs_temp_file.write(self._sample_content("bucketfs"))
+
+    def _read_from_bucketfs(self) -> str:
+        buffer = BytesIO()
+        for chunk in self._bfs_temp_file.read():
+            buffer.write(chunk)
+        return buffer.getvalue().decode("utf-8")
+
+    def _write_to_db_table(self):
+        self.db_table_proxy().insert(self._sample_content("table-insert"))
```

Comment:
ahm, you need to construct an insert statement and return this as part of the Continue of the start method

Reply:
Will be addressed in separate PR on extended scenario.